### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by vishy0001** [shahriarsajeeb/Becodemy-Mobile-App#1]
After cloning the project follows the steps written on github page 1.Npm start and other steps also ;but multiple errors are coming while running this.Can you make 10 min video in which starting from  cloning and running the project in your Andorid and iOS device and various feature it includes and al steps which need to follow to run the same.

**Comment by Maccroe** [comment]
Yes make an tutorial how to clone and run this project on android because in android there are many problems like [com.google.android.gms.common.api.apiexception: developer_error] when trying to login with github

**Comment by kinza7124** [comment]
Yes please @shahriarsajeeb
